src/Native/System.Native/pal_networking.cpp

```diff
@@ -21,16 +21,14 @@ static_assert(PAL_TRY_AGAIN == TRY_AGAIN, "");
 static_assert(PAL_NO_RECOVERY == NO_RECOVERY, "");
 static_assert(PAL_NO_DATA == NO_DATA, "");
 static_assert(PAL_NO_ADDRESS == NO_ADDRESS, "");
-static_assert(sizeof(uint8_t) == sizeof(char), ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
+static_assert(sizeof(uint8_t) == sizeof(char),
+              ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
 
 static int IpStringToAddressHelper(const uint8_t* address, const uint8_t* port, bool isIPv6, addrinfo*& info)
 {
     assert(address != nullptr);
 
-    addrinfo hint = {
-        .ai_family = isIPv6 ? AF_INET6 : AF_INET,
-        .ai_flags = AI_NUMERICHOST | AI_NUMERICSERV
-    };
+    addrinfo hint = {.ai_family = isIPv6 ? AF_INET6 : AF_INET, .ai_flags = AI_NUMERICHOST | AI_NUMERICSERV};
```

Comment:
From my initial investigation, it's all-or-nothing (see the comment below). We can definitely change this rule if we so choose, but let's chat about it in a separate issue and just get the code into current compliance; this PR and #3761 combined will make it so any changes we make to the formatting style will happen seamlessly, so it wouldn't be a problem to change this in the future